src/matrix/storage/idb/error.ts

```diff
@@ -34,22 +37,22 @@ export class IDBError extends StorageError {
         if (cause) {
             fullMessage += cause.message;
         }
-        super(fullMessage, cause);
+        super(fullMessage, cause || undefined);
```

Comment:
Because `cause` can be `null`, but the argument expects something that can be `undefined`. This was the least invasive way of getting it to typecheck, I think.

Reply:
I didn't know about this interaction with `undefined` and default arguments. It seems like this is indeed the best approach; I'll switch to using `| null` in `StorageError` and friends. I don't see anywhere that accesses `.cause` though.